utils.py

```diff
@@ -16,5 +16,6 @@
         CSV_FAULTS.append(row)
 
 
-def displayHash(rack: str, faultCondition: str, okCondition: str, tlc: str, suffix: str):
-    return hash(rack) ^ hash(faultCondition) ^ hash(okCondition) ^ hash(tlc) ^ hash(suffix)
+def displayHash(rack: str, faultCondition: str, okCondition: str, tlc: str, suffix: str,
+                prefix: str):
+    return hash(rack) ^ hash(faultCondition) ^ hash(okCondition) ^ hash(tlc) ^ hash(suffix) ^ hash(prefix)
```

Comment:
Was there a problem with faults being overwritten because of the prefix? I naively didn't expect it to matter

Reply:
Our xcor, ycor, and quad MGT faults all had the exact same rack (none), faultCondition (8), okCondition (none), tlc (MGT), and suffix (STATMSG) so they were only overwriting each other